lexicons/com/atproto/moderation/defs.json

```diff
@@ -41,6 +41,25 @@
     "reasonAppeal": {
       "type": "token",
       "description": "Appeal: appeal a previously taken moderation action"
+    },
+    "reasonContext": {
+      "type": "string",
+      "description": "Tag describing an aspect or context of a moderation subject relevant to a report.",
+      "knownValues": [
+        "image",
+        "video",
+        "link",
+        "interaction"
+      ]
+    },
+    "subjectType": {
```

Comment:
This doesn't seem to be used anywhere? I'm guessing the incoming report would set this or are we going to infer from the subject itself?

Reply:
the client app would use this to decide which subscribed labelers are in-scope for any particular report. it is declared in `app.bsky.labeler.service`. same with `subjectCollections` (array of NSIDs).

neither need to be included in reports because the subject type (DID vs strongRef-with-ATURI) contains the same info.